jwst/guider_cds/guider_cds.py

```diff
@@ -1,31 +1,16 @@
-#! /usr/bin/env python
-#
-#  guider_cds.py
-
 import numpy as np
 import logging
 
 from stdatamodels.jwst import datamodels
 
-from ..lib import reffile_utils
+from jwst.lib import reffile_utils
 
 log = logging.getLogger(__name__)
 log.setLevel(logging.DEBUG)
 
-# When an appropriate ref file for the gain (read noise) can not be retrieved
-#  for the calculation of the variances, and then the ERR array constant
-#  value equal to the mean of the SCI values in this accessible ref file will
-#  be used:
-DEFAULT_GAIN = 1.9713863
-DEFAULT_READNOISE = 21.0
-DEFAULT_GAIN_FILE = "jwst_fgs_gain_0010.fits"
-DEFAULT_READNOISE_FILE = "jwst_fgs_readnoise_0000.fits"
-
 
-def guider_cds(model, gain_model=None, readnoise_model=None):
+def guider_cds(model, gain_model, readnoise_model):
```

Comment:
I like that you made these required (assuming there were no meaningful use-cases where they could be set to None), but it looks like that caused some unit tests to fail